### PR TITLE
fix(chat): scroll through glyphs/emoji

### PR DESCRIPTION
**Pull request by vimercati-samir**
What this PR does 📖
Fix scroll through glyphs/emoji

Which issue(s) this PR fixes 🔨
AP-889

Special notes for reviewers 🗒️

Additional comments 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: components/views/chat/enhancers/glyphs/Glyphs.less [hunk 1]
```diff
@@ -15,18 +15,21 @@
...
+    /* overflow-y: auto;
     padding-bottom: 80px;
     &::-webkit-scrollbar {
       width: 5px;
     }
     &::-webkit-scrollbar-thumb {
       background-color: var(--flair-color);
       border-radius: 6px;
-    }
+    } */
```
[the remark this answers is not in the document]
Comment (vimercati-samir) [reply components/views/chat/enhancers/glyphs/Glyphs.less]: removed it in commit a60d46b8d74545807866836d6f658d9003cbea66

**Review by josephmcg — CHANGES_REQUESTED** [review]
This breaks the virtual list. change the value of :keeps to something lower. I believe it needs a fixed height in order to work, that's why I added a css scrollbar instead
https://github.com/tangbc/vue-virtual-scroll-list

**Comment by vimercati-samir** [comment]
> This breaks the virtual list. change the value of :keeps to something lower. I believe it needs a fixed height in order to work, that's why I added a css scrollbar instead https://github.com/tangbc/vue-virtual-scroll-list


https://user-images.githubusercontent.com/52865716/156594654-f92f1424-3248-4777-9099-d9cc3b0f5233.mp4

I think it's working properly, I manually added more sticker packs (38 to be precise) and I left the "keeps" parameter to 15 and as you can see in the DOM on the right side of the video the list items are always 15 and on scroll they update, is this behavior correct and perhaps it didn't seem to work because the "keeps" threshold was higher than the number of sticker packs or is still wrong?

**Review by phillsatellite — APPROVED** [review]

**Review by josephmcg — APPROVED** [review]
I only tested changing :keeps to 4 with the usual number of packs. It breaks in that instance.
However, that's not a realistic use case for our app. We would likely keep it higher, more similar to your example. I guess it's alright then.

Please confirm that it's loading every pack that you added

**Review by stavares843 — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/29093946/156681976-fff9c7ee-a91d-46e2-903b-2844b6fe62b7.mov


the scroll is more fluid on dev compared with this branch

**Comment by vimercati-samir** [comment]
> any news? 🔨

fixing and closing it now

**Comment by vimercati-samir** [comment]
> Gravacao.do.ecra.2022-03-04.as.01.30.44.mov 
> the scroll is more fluid on dev compared with this branch
> 
> Chrome, macOS

Should be resolved in commit db62e22bf598bb2d1d11529671fc75561eacc076

**Review by josephmcg — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**